draft-vvv-webtransport-quic.md

```diff
@@ -237,21 +241,20 @@ The user agent MUST send the "Origin" field.  The "Origin" field MUST be set to
 the origin of the client initiating the connection, serialized as described in
 the "serializing a request origin" section of [FETCH].
 
+The server indicates that it accepts the origin by sending its header block and
+a FIN on the stream 0.  If the server does not accept the origin, it MUST close
+the connection.
+
 ### Path Field  {#path-field}
 
 In order to allow multiplexing multiple application on the same host-port tuple,
 QuicTransport allows specifying extra routing information in the path component
-of the URI.  That component is communicated using the "Path" field in the client
-indication:
+of the URI.  That component is communicated using the "Path" header:
 
 Name:
 
 : Path
```

Comment:
Shouldn't this be lower-cased?

Reply:
I'm following the HTTP convention here where headers look like "Content-Type", even though on the wire, HTTP/2 and HTTP/3 encode those as all-lowercase.